src/python/pants/notes/1.21.x.rst

```diff
@@ -0,0 +1,214 @@
+1.21.x Stable Releases
+======================
+
+This document describes releases leading up to the ``1.21.x`` ``stable`` series.
+
+1.21.0rc0 (09/23/2019)
+----------------------
+
+New Features
+~~~~~~~~~~~~
+
+* Add "did you mean" functionality to runtime Param mismatches. (#8315)
```

Comment:
Think this should be refactor. It only impacts rule authors during development.

Reply:
We'll, rule authors are regular users if we did it right.

Reply:
=john